src/test/regress/input/multi_view.source

```diff
@@ -0,0 +1,164 @@
+--
+-- MULTI_VIEW
+--
+
+-- This file contains test cases for view support. It verifies various
+-- Citus features: simple selects, aggregates, joins, outer joins
+-- router queries, single row inserts, multi row inserts via insert
+-- into select, multi row insert via copy commands.
+
+CREATE TABLE lineitem_hash (
+	l_orderkey bigint not null,
+	l_partkey integer not null,
+	l_suppkey integer not null,
+	l_linenumber integer not null,
+	l_quantity decimal(15, 2) not null,
+	l_extendedprice decimal(15, 2) not null,
+	l_discount decimal(15, 2) not null,
+	l_tax decimal(15, 2) not null,
+	l_returnflag char(1) not null,
+	l_linestatus char(1) not null,
+	l_shipdate date not null,
+	l_commitdate date not null,
+	l_receiptdate date not null,
+	l_shipinstruct char(25) not null,
+	l_shipmode char(10) not null,
+	l_comment varchar(44) not null );
+
+SET citus.shard_count to 4;
+SET citus.shard_replication_factor to 1;
+
+SELECT create_distributed_table('lineitem_hash', 'l_orderkey');
+
+\copy lineitem_hash FROM '@abs_srcdir@/data/lineitem.1.data' with delimiter '|'
+\copy lineitem_hash FROM '@abs_srcdir@/data/lineitem.2.data' with delimiter '|'
```

Comment:
Hm. Can't we move this to one central file? We now create, load, dump this in multiple places. And all of that requires us to use .source files, whic hare much more painful to maintain.  So please move lineitem_hash etc from all tests into a multi_create_table.sql/multi_load_data.source.